pkg/addons/translate.go

```diff
@@ -72,6 +98,7 @@ var Translations = map[string][]struct {
 	NestedPath  jsonpath.Path
 	NestedFlags NestedFlags
 	Template    string
+	F           func(*acsapi.OpenShiftManagedCluster) string
```

Comment:
You can still use the derived lbc prefix without this, no?

Reply:
as a function to get value?

Reply:
I think this is fine, nvmd